### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by joaquincollado**
Created base for react induction target

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,8055 @@
+{
```
Comment (mato2593) [inline package-lock.json:1]: You should have only one of `package-lock.json` or `yarn.lock`. If you are going to use yarn, then you have to delete `package-lock.json`, and if you are going with npm, then it's the other way around.

**Comment by juan-apa** [comment]
Maybe try adding a `.gitignore` file that excludes the `node_modules` folder in this initial commit, so that you don't have to omit that folder manually in the following commits.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "react-target",
+  "description": "React Target Induction",
+  "scripts": {
+    "start": "webpack-dev-server --config webpack.config.dev.js --port 3000"
+  },
+  "dependencies": {
+    "html-webpack-plugin": "^4.3.0",
```
Comment (bernabe9) [inline package.json:8]: This is a dev dependency

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { render } from "react-dom";
+
+function Hi() {
```
Comment (bernabe9) [inline src/index.js:4]: use an arrow function instead.

```suggestion
const Hi = () => <p>Hi.</p>
```

**Pull request merged**